### PR TITLE
Add Artifactory and Xray versions in CHANGELOG.md

### PR DESCRIPTION
**Pull request by danielmkn**
- Do not cancel pending run on PR modification, because it will cancel the valid job after Pipeline will commit CHANGELOG modification
- Update Go version to 1.18
- Limit the PR title by 100 characters

### CODE REVIEW

File: TFproviderTest.yml [hunk 1]
```diff
@@ -63,12 +63,12 @@ pipelines:
...
             - echo "Helm chart version "${rt_helm_chart_version}
-            - add_run_variables artifactory_version=$(helm search repo | grep "${ARTIFACTORY_DISTRIBUTION} " | awk '{$1=$1};1' |  cut -f3 -d " ")
-            - echo "Artifactory version "${artifactory_version}
+            - add_run_variables ARTIFACTORY_VERSION=$(helm search repo | grep "${ARTIFACTORY_DISTRIBUTION} " | awk '{$1=$1};1' |  cut -f3 -d " ")
+            - echo "Artifactory version "${ARTIFACTORY_VERSION}
             - add_run_variables xray_helm_chart_version=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f2 -d " ")
             - echo "Xray Helm chart version "${xray_helm_chart_version}
-            - add_run_variables xray_version=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f3 -d " ")
-            - echo "Xray version "${xray_version}
+            - add_run_variables XRAY_VERSION=$(helm search repo | grep "/xray" | awk '{$1=$1};1' |  cut -f3 -d " ")
```
Comment (chb0github) [inline TFproviderTest.yml:70]: What format does this take?

**Review by chb0github — APPROVED** [review]

File: TFproviderTest.yml (hunk 1 above)
Comment (danielmkn) [reply TFproviderTest.yml:70]: The requirements came from [Slack documentation](&amp)

Comment (danielmkn) [reply TFproviderTest.yml:70]: it's just text format, if I understood your question correct

**Comment by chb0github** [comment]
I meant the actual text format. Of course it's text, but what form

**Pull request merged**